LDAP/Init.hsc

```diff
@@ -106,6 +107,17 @@ ldapSimpleBind ld dn passwd =
            return ()
                          )))
 
+{- | Bind with the SASL EXTERNAL mechanism. -}
+ldapExternalSaslBind :: LDAP   -- ^ LDAP Object
+                     -> String -- ^ Authorization identity (UTF-8 encoded; pass "" to derive it from the authentication identity)
```

Comment:
Good that you solved naming problem :)
UTF-8 and `String` sound odd together, though